# HG changeset patch
# User Vytautas Janƒçauskas <unaudio@gmail.com>
# Date 1378819637 -10800
#      Tue Sep 10 16:27:17 2013 +0300
# Node ID 3c35ee570366724667a78d360b980fe886eedbf3
# Parent  febd874d363dab6f0a9182b85d1eab692bbc8349
configure.ac: fix a typo where magick flags were used instead of sndfile ones

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -938,19 +938,19 @@ CPPFLAGS="$save_CPPFLAGS"
 
 SNDFILE_CPPFLAGS=`$PKG_CONFIG --cflags-only-I sndfile`
 SNDFILE_LDFLAGS=`$PKG_CONFIG --libs-only-L sndfile`
 SNDFILE_LIBS=`$PKG_CONFIG --libs-only-l sndfile`
 
 CPPFLAGS="$SNDFILE_CPPFLAGS $CPPFLAGS"
 LIBS="$SNDFILE_LDFLAGS $SNDFILE_LIBS $LIBS"
 
-AC_SUBST(MAGICK_CPPFLAGS)
-AC_SUBST(MAGICK_LDFLAGS)
-AC_SUBST(MAGICK_LIBS)
+AC_SUBST(SNDFILE_CPPFLAGS)
+AC_SUBST(SNDFILE_LDFLAGS)
+AC_SUBST(SNDFILE_LIBS)
 
 ### Check for either of Graphics/ImageMagick++ libraries
 
 AC_ARG_WITH([magick],
   [AS_HELP_STRING([--with-magick=LIB],
     [select library to use for image I/O (options: GraphicsMagick(default) or ImageMagick)])],
   [magick="$withval"],
   [magick="GraphicsMagick"])
